tests/probes/symlink/test_offline_mode_symlink.sh

```diff
@@ -45,7 +45,12 @@ function test_offline_mode_symlink {
 
     bash ${srcdir}/test_offline_mode_symlink.xml.sh "" > "$DF"
     export OSCAP_PROBE_ROOT="$tmpdir"
-    $OSCAP oval eval --results $RF $DF
+    if test -x "$OSCAP_CHROOTABLE_EXEC"; then
```

Comment:
Can we reuse `require "$OSCAP_CHROOTABLE_EXEC" || return 255` or create another special function `require_offline`? And then also we could just replace $OSCAP with OSCAP=$OSCAP_CHROOTABLE in all related tests.